model/http-common.yaml

```diff
@@ -63,6 +63,28 @@ groups:
           conditionally_required: If and only if one was received/sent.
         brief: '[HTTP response status code](https://tools.ietf.org/html/rfc7231#section-6).'
         examples: [200]
+      - id: error.reason
+        brief: 'Low-cardinality error reason.'
+        type:
+          allow_custom_values: true
+          members:
+            - id: empty
+              value: ""
+              brief: 'No error. Default value.'
+            - id: other
+              value: "_OTHER"
+              brief: 'Any error reason instrumentation does not define custom value for.'
+        requirement_level:
+          conditionally_required: If request or response has ended prematurely. When missing, the value is assumed to be `empty`.
```

Comment:
Do you have any ideas how the term 'prematurely' could be defined more clearly?
For a request it would be before a complete response has been received from the peer and for a response it would be before sending the response was completed, right?

Reply:
I tried to define it in the note, please take a look.
Essentially, it has ended prematurely if something prevented the client from reading the response (or instrumentable portion of it)  or prevented the server from sending the response.